src/cmd/_main.cpp

```diff
@@ -35,17 +35,34 @@
 
 #include <chrono>
 
+void print_copyright()
+{
+	printf("\n"
+		" ******************************************\n"
+		" **        Converter PMX to PIX          **\n"
+		" **       Copyright (C) 2021 mwl4        **\n"
```

Comment:
The copyright year should stay the same.

Reply:
Already answered above. I don't see any problems. I wanted the best.